### ISSUE TITLE
Footer Section 

### ISSUE THREAD

**Issue by parthpathakpp1** [Bookingjini-Labs/bookingjini-icons#227]
### Is your feature request related to a problem? Please describe.

There is no Footer Section in the website . Footer Section is needed . 

### Describe the solution you'd like

I will create a Footer Section which will contain all the necessary things in it . 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by parthpathakpp1** [comment]
Can you assign me this issue under GSSOC'23.

**Comment by fantasticbuddy** [comment]
I went through the website and noticed that the footer-section needs some improvement.  I tried to make it more user-friendly by adding some content as well as CSS. This is fully respomsive.

Before -
![Screenshot 2023-06-06 022232](https://github.com/Bookingjini-Labs/bookingjini-icons/assets/114503561/06fb424e-ebbc-423c-a485-4e308e18ea26)

After -
![Screenshot 2023-06-06 023710](https://github.com/Bookingjini-Labs/bookingjini-icons/assets/114503561/2a0bd413-4250-41e2-b753-d13c75f79874)

If you like this change, kindly assign me this issue under GSSOC'23
@AshutoshDash1999 
Thankyou.
